### ISSUE TITLE
[Bug] `$h.provideHover`: ERR Cannot read properties of undefined (reading 'property')

### ISSUE THREAD

**Issue by ollema** [willofindie/vscode-cssvar#87]
**Describe the bug**
I'm getting this strange error when trying to hover over a variable.

Autocomplete is also not working. Only have the Svelte extension, the PostCSS extension and this extension enabled.

The error leads me to believe that it is indeed this extension that is failing:
```
ERR Cannot read properties of undefined (reading 'property'): TypeError: Cannot read properties of undefined (reading 'property')
	at $h.provideHover (/Users/s0001325/.vscode/extensions/phoenisx.cssvar-2.4.1/out/extension.js:68:519)
```

I would be very happy to provide more info/context but I don't know where to start. I have attached a small screen recording of what is happening.

**To Reproduce**
1. Enable extension
2. Hover on css variable

**Expected behavior**
Autocomplete and hover should work

**Screenshots**

https://user-images.githubusercontent.com/31876997/197547077-6e4d1625-30d7-4a32-b983-2c1a75b14943.mov

**Details (please complete the following information):**
 - OS: macOS Monterey 12.6
 - VSCode version: 1.72.2
 - Extension version 2.4.1

**Comment by phoenisx** [comment]
Thanks, @ollema for raising this issue.
_Your attached video file is in `.mov` format unsupported on Firefox 😅_ 

Anyways I tried replicating this issue in `examples/css-in-node-modules` example, but couldn't.

I think this issue happens only in `.svelte` files with `.postcss` styles in it.

Can u help me with the following:
- Are you using any `postcss` plugins/syntax parsers for your styles?
- What happens if you change `style lang` to `css`?

Anyways it would be great if you can share a repro of this issue. Or else give me some time to setup and recreate this bug to figure out the issue 👍🏽

**Comment by ollema** [comment]
for some reason, this started working today. I have closed and reopened vscode a dozen time but today it started working. I will close this for now and re-open with a repro if it happens again

**Comment by phoenisx** [comment]
There was a recent release `v2.4.0` which had a bug, that I fixed in `v2.4.1`. There's a high possibility that the error you got relates to a previous bug.

I will open a new issue, pointing to this one, to handle such errors more gracefully 👍🏽 

Thanks for your support!
